cypher/cypher-docs/src/docs/dev/ql/administration/show-databases-columns.asciidoc

```diff
@@ -0,0 +1,18 @@
+[options="header" cols="2l,4,1,2"]
+|===
+|Column |Description|Included by default|Notes
```

Comment:
I kinda dislike both of those, the 'Default output' icon is not clear what it means, and the constranits has column full of ticks, which doesn't really show anything because everything it ticked. So yes, maybe.

Reply:
I don't like how the icon looks either, prefer either your version or the constraint/index one. Might even be good to have a day of clean-up card to update those places to be more like this one...